CHANGES.rst

```diff
@@ -11,6 +11,12 @@ assign_wcs
   uncaught exception when SIP approximation fails to reach desired
   accuracy. [#7036]
 
+tweakreg
```

Comment:
Needs to be moved up to the top section under the "1.7.3" section heading.

Reply:
I can't do this next 2 weeks. Could you please fix this?

Reply:
Will do.